default.py

```diff
@@ -28,10 +29,14 @@
 cookie_file = os.path.join(ADDON_PROFILE, 'cookie_file')
 username = addon.getSetting('email')
 password = addon.getSetting('password')
-if addon.getSetting('debug') == 'false':
-    debug = False
-else:
-    debug = True
+debug_cmd = {  # determine if debug logging is activated in kodi
+    'jsonrpc': '2.0',
```

Comment:
Is JSONRPC the only interface we can use? Or is there something along the lines of `xbmc.getSetting('debuglevel')`?

If JSONRPC only, can we use it for all settings acquisition? It'd just be nice to not have two different methods.